website/dbt-global-variables.js

```diff
@@ -0,0 +1,15 @@
+exports.dbtVariables = {
+  dbtCore: {
+    name: "dbt Core",
+    versions: [
+      {
+        "name": "dbt Core Lite",
+        "version": "1.0"
+      },
+      {
+        "name": "dbt Lite",
+        "version": "0.20"
+      }
+    ]
+  } 
```

Comment:
@runleonarun what sorts of things do we think we'll be defining in here? At EP we had a Brand variable for our automated emails and it wound up being more of a pain than a benefit, because we did a bad job of guessing what components were going to be renamed over time.

Made up example - if we had `dbtCloud` as a catchall variable for the SaaS product, and then we come up with a fun name for the IDE like `The Power Console` (🤢 ) then we still have to go and split out the new component into its smaller chunks... 

Slightly concerned at the thought that we might have recency bias around "what if the company name changes again" which will hopefully not happen again any time soon! 

100% open to being talked out of it if you've found them useful in a past life though

Reply:
@joellabes It is a good way to easily rename a feature without the hassle of missing many instances like we would with search & replace. But more importantly, it's a way to standardize on spelling and capitalization of a feature or any name in dbt. A great example is `dbt Labs` If we use a product variable for dbt, then it will decrease the chance there will be typos (dBt labs</gasp>). 

Product variables will be really important as we scale and more folks are writing docs. For example, do we say dbt Core v1 or dbt Core v1.0 or dbt Core Version 1.0 ?!? Product variables will also be useful when we're ready to translate. The translators can ignore everything in a variable OR translate in one place, consistently.